.github/workflows/contracts-tests.yml

```diff
@@ -181,10 +181,13 @@ jobs:
           lcov --remove lcov_merged.info -o lcov_merged.info
             'test/*'
             'script/*'
+            'src/Dependencies/Ownable.sol'
+            'src/Zappers/Modules/Exchanges/UniswapV3/UniPriceConverter.sol'
             'src/Interfaces/*'
```

Comment:
This is not a result of the PR, but `lcov` is complaining that (and failing because) `'src/Interfaces/*'` is an unused pattern. Should we just remove it?

Reply:
https://github.com/liquity/bold/actions/runs/12676876780/job/35330995484?pr=683#step:8:11

Reply:
Weird. Yes, I’ll remove it